README.md

```diff
@@ -34,6 +34,17 @@ Bazelisk currently understands the following formats for version labels:
 
 In the future we will add support for building Bazel from source at a given commit.
 
+## How does Bazelisk download release from a fork?
+
+It uses a simple algorithm:
+- If the environment variable `USE_BAZEL_REMOTE` is set, it will use the fork name specified in the value.
+- Otherwise, if a `.bazelremote` file exists in the current directory or recursively any parent directory, it will read the file and use the the fork name specified in it.
```

Comment:
Instead of using a separate file, what do you think of adding a "fork/" prefix to the .bazelversion file (and general version label parsing) instead? So users could do this:

echo "philwo/latest" > ~/.bazelversion
or
USE_BAZEL_VERSION="philwo/0.28.1"

It would be backwards compatible, too.

Reply:
This is a good idea too.